classes/components/listPanels/PKPSubmissionsListPanel.inc.php

```diff
@@ -51,6 +52,10 @@ public function getConfig() {
 		$config['itemsMax'] = $this->itemsMax;
 
 		// URL to add a new submission
+		if ($context->getData('disableSubmissions')) {
+			$config['showButton'] = false;
```

Comment:
`showButton` needs to be updated to `allowSubmissions` here.

Reply:
Sorry, I forgot to commit that one, should be fixed now.